dashboard/app/views/levels/_open_response.html.haml

```diff
@@ -0,0 +1,16 @@
+:javascript
+  function getResult() {
+    return {
+      response: $('.response').val(),
+      result: true
+    };
+  }
+
+#open-response
+  %div= ActionView::Base.new.render(inline: @level.markdown_instructions, type: :md)
```

Comment:
Ah cool, `render` works!  I had copied and pasted this from the old view.  Fixed in 9080503dda243c6698b5fd77aeced6062ba5b5c0.